torch/csrc/jit/passes/quantization.cpp

```diff
@@ -752,6 +752,31 @@ bool isWeightOfConvOrLinear(Value* v) {
   return result;
 }
 
+// Go through the CallMethod graph to check if the value is Weight.
+c10::optional<bool> isWeight(Module& module, Value* v) {
+  if (isWeightOfConvOrLinear(v)) {
+    return true;
+  }
+  c10::optional<bool> result;
+  auto* self = v->owningGraph()->inputs()[0];
+  for (const Use& u : v->uses()) {
+    Node* n = u.user;
+    if (n->kind() == prim::CallMethod) {
+      auto m = getInvokedModule(module, n, self);
+      auto g = m.get_method(n->s(attr::name)).graph();
+      auto call_method_result = isWeight(m, g->inputs()[u.offset]);
+      if (result) {
```

Comment:
I think this has to be `result.has_value()`, since if result is false this might not hold either